tests/bluetooth/bsim_bt/bsim_test_mesh/src/test_provision.c

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright (c) 2021 Lingao Meng
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include "mesh_test.h"
+
+#include <sys/byteorder.h>
+
+#define LOG_MODULE_NAME mesh_prov
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(LOG_MODULE_NAME);
+
+/*
+ * Provision layer tests:
+ *   Tests both the provisioner and device role in various scenarios.
+ */
+
+/* Timeout semaphore */
+static struct k_sem prov_sem;
+
+static void test_device_init(void)
+{
+}
+
+static void test_provisioner_init(void)
+{
+}
```

Comment:
Need to initialize the test roles, like in the other tests. This is what's failing in the screenshot - provisioner doesn't have a `cfg` set, but we're trying to read it in `bt_mesh_test_setup`.